style.css

```diff
@@ -0,0 +1,119 @@
+html, body {
+  height: 100%;
+  width: 100%;
```

Comment:
There is a display "glitch" (website sliding slightly) on mobile that you could fix by adding width: Auto, instead of 100%.